Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -1490,12 +1490,13 @@ def return_outputs(readable_output, outputs=None, raw_response=None):
     }
     # Return 'readable_output' only if needed
     if readable_output and not outputs and not raw_response:
-        demisto.results(readable_output)
+        return_entry["Contents"] = readable_output
+        return_entry["ContentsFormat"] = formats["text"]
+        demisto.results(return_entry)
```

Comment:
remove. You have the results call in the end. make sure the unit test check that results is called only once.